src/github3/repos/branch.py

```diff
@@ -121,15 +121,19 @@ def protect(self, enforcement=None, status_checks=None):
 
     @decorators.requires_auth
     def unprotect(self):
-        """Disable force push protection on this branch."""
-        edit = {'protection': {'enabled': False}}
-        resp = self._patch(self._api, json=edit,
-                           headers=self.PREVIEW_HEADERS)
-        json = self._json(resp, 200)
-        if self._boolean(resp, 200, 404):
-            self._update_attributes(json)
-            return True
-        return False
+        """Disable force push protection on this branch.
+
+        See: https://git.io/fxctW
```

Comment:
Is there a reason to use a URL shortener here? I'm not generally a fan of obscuring a thing we're asking people to click on.

Reply:
My reason was PEP8: the URL behind the shortener would otherwise make the docstring violate the line length limit.

Reply:
@jacquerie If you instead write it as:

```
See also:
https://super-long-url-here
```

Flake8 will ignore it by default.

Reply:
Uh, I didn't know that! Good to know, will do.